docs/releases.md

```diff
@@ -40,11 +40,10 @@ Otherwise, we typically release every two weeks:
 
 ## Kubernetes Compatibility Matrix
 
-| Argo Workflows \ Kubernetes | 1.26 | 1.27 | 1.28 | 1.29 | 1.30 |
-|-----------------------|------|------|------|------|------|
-| **3.5**               | `✓` | `✓` | `✓` | `✓` | `?` |
-| **3.4**               | `✓` | `✓` | `?` | `?` | `?` |
-| **3.3**               | `?` | `?` | `?` | `?` | `?` |
+| Argo Workflows \ Kubernetes | 1.28 | 1.29 | 1.30 |
+|-----------------------------|------|------|------|
+| **3.5**                     | `✓`  | `✓`  | `✓`  |
```

Comment:
hmmm I wonder if we should be changing this table given that we're not backporting these changes